libraries/SITL/SITL.cpp

```diff
@@ -234,6 +234,8 @@ const AP_Param::GroupInfo SITL::var_info3[] = {
     AP_GROUPINFO("THML_SCENARI",12, SITL,  thermal_scenario, 0),
     
     AP_GROUPINFO("GPS2_HDG",      13, SITL,  gps_hdg_enabled[1], 0),
+    // Buyoancy for submarines
+    AP_GROUPINFO("BUOYANCY",      14, SITL,  buoyancy, 1),
```

Comment:
There is a way to make parameters vehicle-specific. @Williangalvani can you try that? @peterbarker may have a hint.

Reply:
Aha, try this https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Baro/AP_Baro.cpp#L127